scripts/indexes.js

```diff
@@ -41,6 +41,9 @@ db.patches.ensureIndex({ "branch" : 1, "create_time" : 1 })
 db.patches.ensureIndex({ "version" : 1 })
 db.patches.ensureIndex({ "author" : 1, "create_time" : 1 })
 
+//======patch_intents======//
+db.patch_intents.ensureIndex({ "msg_id" : 1 }, { "unique" : true, "partialFilterExpression" : { "intent_type" : "github"}})
```

Comment:
why use a filter here? presumably even when there are other kinds of intents we'll want to index on that too. this seems like a slightly premature optimization.

Reply:
I did it because msg_id is an attribute only on the github style intents, but I have no problem removing it.